src/state/index.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * State management for Lonboard.
+ *
+ * Two types of state:
+ * - Client-only: UI state that never syncs with Python (Zustand)
+ * - Python-synced: State that bidirectionally syncs with Python (Backbone)
```

Comment:
Maybe just add a tiny note that `store.ts` holds the client-only state


```suggestion
 * - Client-only: UI state that never syncs with Python (Zustand). See `store.ts`
 * - Python-synced: State that bidirectionally syncs with Python (Backbone). See `python-sync.ts`
```